toolset/benchmark/benchmarker.py

```diff
@@ -561,6 +561,7 @@ def exit_with_code(code):
             sudo service postgresql restart
             sudo service cassandra restart
             /opt/elasticsearch/elasticsearch restart
+   	    sudo /etc/init.d/apache2 stop
```

Comment:
Is there a reason why this is a stop and not a restart like the other services in this function?